python/tvm/relay/frontend/sklearn.py

```diff
@@ -0,0 +1,154 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name, import-self, len-as-condition, unused-argument, too-many-lines
+# pylint: disable=import-outside-toplevel
+
+import numpy as np
+import tvm
+from tvm.ir import IRModule
+
+from ... import nd as _nd
+from .. import analysis
+from .. import expr as _expr
+from .. import function as _function
+from .. import op as _op
+from .. import vision as _vision
+
+from ..function import Function
+from ..expr import Call, Let
+from ..expr import If, Tuple, TupleGetItem
+from ..expr import RefCreate, RefRead, RefWrite
+from ..expr_functor import ExprFunctor
+from ..adt import Match, Clause
+
+from .common import AttrCvt, Renamer, ExprTable
+from .common import get_relay_op, new_var, infer_shape, infer_channels
+from .common import infer_type, get_name
+from .common import infer_value as _infer_value
+from .common import infer_value_simulated as _infer_value_simulated
+
+
+def _SimpleImputer(op, inexpr, dshape, dtype, columns=None):
+    boolean_mask = _op.isnan(inexpr)
+    fill_col = _op.const(np.array(op.statistics_, dtype=dtype))
+    input_shape = _op.shape_of(inexpr)
+    reps = _op.take(input_shape, _op.const([0]))
+    reps = _op.concatenate([reps, _op.const([1])], axis=0)
+
+    fill_val = _op.tile(fill_col, reps=reps)
+    indices =_op.const(np.arange(len(op.statistics_)))
+    fill_val = _op.take(fill_val, indices=indices, axis=1)
+
+    ret = _op.where(boolean_mask,
+                    fill_val,
+                    inexpr)
+    
+    return ret
+
+def _RobustImputer(op, inexpr, dshape, dtype, columns=None):
+    if columns: 
+        column_indices = _op.const(columns)
+        inexpr = _op.take(inexpr, indices=column_indices, axis=1)
+
+    if not op.mask_function:
```

Comment:
Is op.mask_function a boolean? If yes, then its ok. Otherwise, `if op.mask_function is not None`

Reply:
mask_functrion will be None when it is undefined.